react/error.tsx

```diff
@@ -62,84 +53,9 @@ class ErrorPage extends Component {
   }
 
   public render() {
-    return (
-      <div className="h-100 flex flex-column mh6 mh0-ns error-height pt3 pt10-ns">
-        <div>
-          {this.renderErrorInfo()}
-          {window.__ERROR__ && this.renderErrorDetails(window.__ERROR__)}
-        </div>
-      </div>
-    )
-  }
-
-  private renderErrorInfo = () => {
-    const date = new Date()
-
-    return (
-      <div className="flex justify-center-ns flex-row-ns flex-column-reverse h-auto-ns pt0-ns pb8">
-        <div className="mr9-ns mr0">
-          <div className="f2 c-on-base">Something went wrong</div>
-          <div className="f5 pt5 c-on-base lh-copy">
-            <div>There was a technical problem loading this page.</div>
-            <div>Try refreshing the page or come back in 5 minutes.</div>
-          </div>
-          <div
-            className="f6 pt5 c-muted-2"
-            style={{ fontFamily: 'courier, code' }}
-          >
-            <div>ID: {window.__REQUEST_ID__}</div>
-            <div className="f6 c-muted-2 lh-copy fw7">{date.toUTCString()}</div>
-          </div>
-          <div className="pt7">
-            <button
-              className={
-                'bw1 ba fw5 ttu br2 fw4 v-mid relative pv4 ph6 f5 ' +
-                (this.state.enabled
-                  ? 'bg-action-primary b--action-primary c-on-action-primary hover-bg-action-primary hover-b--action-primary hover-c-on-action-primary pointer'
-                  : 'bg-disabled b--disabled c-on-disabled')
-              }
-              disabled={!this.state.enabled}
-              onClick={() => {
-                window.location.reload()
-              }}
-            >
-              Refresh
-            </button>
-          </div>
-        </div>
-        <div>
-          <img
-            src={ErrorImg}
-            className={`${style.imgHeight} pb6 pb0-ns`}
-            alt=""
-          />
-        </div>
-      </div>
-    )
-  }
+    if (window.__ERROR__) console.error(window.__ERROR__)
```

Comment:
Do you know if we can capture this error using Sentry? IMO we should avoid using console logs in prod

Reply:
Yes! We can! In fact, I think we already do :thinking:

Reply:
Yep, we do! Take a look at lines 34-52

![image](https://github.com/user-attachments/assets/b80c9b2c-001a-42f6-9981-7f4de087ae8d)

Reply:
I think we discuss it further, currently we show this error directly on the UI, since we won't be doing that any longer this was the proposal to simplify debug on error (nowadays people often just print the screen and share it for debug)